File: Conversao Maiusculo e Minusculo/maiusminus.lua
io.write("Digite um texto .....: ")
local texto = io.read()

local textoMaiusculo = string.upper(texto)
local textoMinusculo = string.lower(texto)

io.write("Texto em maiusculo ..: ")
print(textoMaiusculo)

io.write("Texto em minusculo ..: ")
print(textoMinusculo)

File: Conversao Maiusculo e Minusculo/maiusminus.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)

	fmt.Print("Digite um texto .....: ")
	texto, _ := reader.ReadString('\n')
	texto = strings.TrimSpace(texto)

	textoMaiusculo := strings.ToUpper(texto)
	textoMinusculo := strings.ToLower(texto)

	fmt.Println("Texto em maiusculo ..: ", textoMaiusculo)
	fmt.Println("Texto em minusculo ..: ", textoMinusculo)
}
